src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,132 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+/--
+We need an additional technical condition on the norm for the following proofs to work
+-/
+class normed_lattice_add_comm_group' (α : Type*)
+  extends normed_lattice_add_comm_group α :=
+(norm_smul: ∀ a : α, 2*∥a∥ ≤ ∥2•a∥)
+
+instance : normed_lattice_add_comm_group' ℝ :=
+{ norm_smul := by finish,
+  ..real.normed_lattice_add_comm_group, }
+
+variables [topological_space α] [normed_lattice_add_comm_group' β]
+
+-- Can we infer this from https://github.com/leanprover-community/mathlib/blob/f29b0b49badab1bcbe338cb79d102e36e79dce09/src/topology/continuous_function/basic.lean#L114 ?
+instance : partial_order (α →ᵇ β) := partial_order.lift (λ f, f.to_fun) (by tidy)
+
+lemma inf_sub_inf_le (a b c d : β)  : ∥a⊓b - c⊓d∥ ≤ ∥a - c∥ + ∥b - d∥ :=
```

Comment:
Isn't this exactly `norm_inf_sub_inf_le_add_norm`?

Reply:
@RemyDegenne I think you're right! Thanks - that removes the technical condition (which never felt right). I've been working on this so long I must have lost the thread somewhere.